### DISCUSSION TITLE
v3 sessions, Poetry, and more

### DISCUSSION THREAD

**Discussion by bug-or-feature** [ig-python/ig-markets-api-python-library#209] [Announcements]
## 0.0.10

* simplified request headers: `requests.Session` now handles persistence
* adds support for v3 session creation (#157)
* implementation and test for fetch_working_orders() v2 (like PR #187)
* adds support for fetch_historical_prices_by_epic_and_date_range() v2 
* adds support for fetch_open_positions() v2
* dependency management with [Python Poetry](https://python-poetry.org/) (#149)
* more robust handling of IG API rate limits
* support for all the `/history/activity` endpoints
* better test coverage. now 80% for `rest.py` (#38)
* documentation improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/ig-python/ig-markets-api-python-library/releases/tag/0.0.10'>v3 sessions, Poetry, and more</a>.</em>

**Comment by bug-or-feature** [comment]
Hi all, I've pushed a 0.0.10 release today. There's a lot of changes. Any questions, comments, missing stuff - let's chat here. But read the [updated docs](https://trading-ig.readthedocs.io/en/latest/index.html) first

**Reply by RRMXkun** [reply]
Huge release, thank you for all this work. I've updated my repo and will report back
